anitya/templates/project_new.html

```diff
@@ -119,10 +121,19 @@
     if (str == 'custom'){
```

Comment:
I think this (and the if's above) should all be in the plugin code themselves.
As it stands now, you have a modular setup, but then still special-case all plugins in the core.